pymc/distributions/distribution.py

```diff
@@ -279,24 +308,70 @@ class SymbolicRandomVariable(OpFromGraph):
     _print_name: tuple[str, str] = ("Unknown", "\\operatorname{Unknown}")
     """Tuple of (name, latex name) used for for pretty-printing variables of this type"""
 
+    @classmethod
+    def _params_signature(cls):
+        inputs_signature, outputs_signature = cls.signature.split("->")
+        inputs_params_signature = [
+            inp_sig for inp_sig in inputs_signature.split(",") if inp_sig not in ("[rng]", "[size]")
```

Comment:
Maybe a global `SPECIAL_TOKENS = ["[rng]", "[size]"]` in case you want to add/remove/modify these later?

Reply:
I think this whole section is just to remove the special tokens before passing it to `_parse_gufunc_signature` yeah? Couldn't this be done more transparently with a regex, like `core_signature = re.replace(special_token_pattern, cls.signature)`?

Reply:
Cross that bridge when/if we get there?

Reply:
sure

Reply:
> I think this whole section is just to remove the special tokens before passing it to _parse_gufunc_signature yeah? Couldn't this be done more transparently with a regex, like core_signature = re.sub(special_token_pattern, '', cls.signature)?

Needs to be able to get rid of the optional/useless commas, but otherwise I'm not against regex, just not my zone.

Reply:
Done the regex way